.ci/install_kata_image_rust.sh

```diff
@@ -34,7 +35,9 @@ build_rust_image() {
 	go get -d "${osbuilder_repo}" || true
 	pushd "${GOPATH}/src/${osbuilder_repo}/rootfs-builder"
 	export ROOTFS_DIR="${GOPATH}/src/${osbuilder_repo}/rootfs-builder/rootfs"
+	sudo rm -rf "${ROOTFS_DIR}"
 	distro="ubuntu"
+	distro="alpine"
```

Comment:
Still work in progress if not  the line `distro=ubuntu` should be removed? Also  I the distro to be used should be defined in https://github.com/kata-containers/runtime/blob/master/versions.yaml#L113

Reply:
@jcvenegas , should keep ubuntu I think.